kernel/antfs/Makefile

```diff
@@ -0,0 +1,44 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=antfs
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/klukonin/antfs.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2020-02-10
+PKG_SOURCE_VERSION:=b41ba529f6b64b429527e09a06ce0326d5456c05
+PKG_MIRROR_HASH:=dae039c0fe5bf1a2c8c1cca4211d607a4d6f56fc41b38444e2234b40d710d9db
+
+PKG_LICENSE:=GPL-2.0-only
```

Comment:
Based on https://github.com/Sembedded/antfs/blob/master/dir.c , this should be -or-later. No idea if absolutely correct.

Reply:
I think that it's completely correct to use original license.